### PR TITLE
PP-12251: Adds some signposting to some API-only features

### PR DESCRIPTION
**Pull request by NathanD-GDS**
### Context
A lot of the pages in the tech docs have pre-requisites that we don’t really spell out. For example, ‘Take recurring payments’ makes more sense if readers have an understanding of how standard payments work. Similarly, ‘Take a payment’ requires users to have some technical knowledge to understand how APIs work.

### Changes proposed in this pull request
This PR:

* tells services that they'll need a technical team to:
  * take recurring payments 
  * use webhooks
* directs services to payment links if they don't have a tech team
* warns members of the public that the 'Refunding a payment' page is not for them
* adds a request example to a MOTO payment docs page

### CODE REVIEW

**Review by asimms41 — COMMENTED** [review]
Looks great. Just a few suggestions.

File: source/moto_payments/moto_api/index.html.md.erb [hunk 1]
```diff
@@ -15,13 +15,20 @@ Make sure you've [turned on MOTO (telephone or post) payments on your account](/
...
+    ```json
+    {
+      "amount": 14500,
+      "reference" : "12345",
+      "description": "Pay your council tax",
+      "return_url": "https://your.service.gov.uk/payment_complete_page",
+      "moto": true
+    }
+    ```
+2. Go to the `next_url` included in the response to your API request.
```
[the remark this answers is not in the document]
Comment (NathanD-GDS) [reply source/moto_payments/moto_api/index.html.md.erb:29]: I think that the instructions on this page are fairly weak and don't really describe the process as well as they could, so I'm going to take this comment away and look at it as part of another piece of work.

I won't make this change but I'll make sure this gets changed soon.

File: source/payment_flow/index.html.md.erb [hunk 2]
```diff
@@ -7,7 +7,7 @@ weight: 1300
 
 # How GOV.UK Pay works
 
-This section outlines the payment flow for paying users and services using GOV.UK Pay.
+This section outlines the payment flow for paying users and services using GOV.UK Pay. It focuses on services that integrate with the GOV.UK Pay API.
```
[the remark this answers is not in the document]
Comment (asimms41) [reply source/payment_flow/index.html.md.erb:10]: I'm not sure if this is necessary but I was coming to this page cold and thinking what type of services does this mean? I'm guessing the users already know?

File: source/making_payments/index.html.md.erb [hunk 3]
```diff
@@ -9,6 +9,8 @@ weight: 2100
 
 This section tells you how to take a payment using the GOV.UK Pay API.
 
+If you want to take payments using a no-code solution instead, you can use [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/).
```
Comment (charleemg) [inline source/making_payments/index.html.md.erb:12]: what is the purpose of "instead" here? I am wondering if it works the same just with "If you want to take payments using a no-code solution, you can use..."

File: source/recurring_payments/index.html.md.erb [hunk 4]
```diff
@@ -19,6 +21,8 @@ Your service has [additional responsibilities when taking recurring payments](/r
 * telling users when you’re going to take or change payments 
 * letting users amend their recurring payments
 
+It would be helpful to familiarise yourself [the standard GOV.UK Pay payment flow](/payment_flow) before reading about recurring payments.
+
 This section outlines the recurring payment flow, your responsibilities, and how to set up and use agreements to take recurring payments.
```
Comment (charleemg) [inline source/recurring_payments/index.html.md.erb:26]: "This section explains"? I think "outlines" is a bit of a policy word, dangerously close to "sets out" :)

File: source/refunding_payments/index.html.md.erb [hunk 5]
```diff
@@ -7,9 +7,9 @@ weight: 4100
 
 # Refund a payment
 
-You can refund all or part of a payment using the GOV.UK Pay API.
+<%= warning_text('If you are a member of the public trying to get a refund, please contact the government department you made the payment to. This page is for civil servants that take payments with GOV.UK Pay. ') %>
```
Comment (charleemg) [inline source/refunding_payments/index.html.md.erb:10]: I wonder about "trying to get a refund" - it sounds a bit like they're fighting/failing against the government machine? Maybe "If you are a member of the public requesting...?"

**Review by charleemg — APPROVED** [review]
All serving suggestions from me, but please check that typo on recurring_payments line 24

File: source/making_payments/index.html.md.erb (hunk 3 above)
Comment (NathanD-GDS) [reply source/making_payments/index.html.md.erb:12]: Good point. Services can use both - I'll adjust.

File: source/refunding_payments/index.html.md.erb (hunk 5 above)
Comment (NathanD-GDS) [reply source/refunding_payments/index.html.md.erb:10]: Very good point, comrade. I'll use your suggestion.

File: source/recurring_payments/index.html.md.erb (hunk 4 above)
Comment (NathanD-GDS) [reply source/recurring_payments/index.html.md.erb:26]: 'explains' is clearer English - good spot!

**Review by edoardo123-PAY — APPROVED** [review]
Looks good

**Pull request merged**